### DISCUSSION TITLE
REST API for create/update/delete operations?

### DISCUSSION THREAD

**Discussion by Hazelton42** [SCADA-LTS/Scada-LTS#2842] [General]
Was looking at the provided REST yaml, it seems very focused on the GET operations.  Reverse engineering from the UI I can see POST and DELETE operations for creating/deleting data sources, etc.  Is there separate documentation for these operations?  

Second question: Is there a way to bulk create data points or create a data source with predefined data points from a template?

**Comment by Limraj** [comment]
Hi @Hazelton4 ,
I invite you to this issue: #2518

Best regards.
